packages/react-components/src/templates/AnalyticsPageHeader.tsx

```diff
@@ -34,10 +28,16 @@ const AnalyticsPageHeader: React.FC = () => (
         </Paragraph>
       </div>
       <TabNav>
-        <TabLink href={analytics({}).$}>
-          <span css={iconStyles}>
-            <LeadershipIcon color={charcoal.rgb} />
-          </span>
+        <TabLink
+          href={analytics({}).productivity({ metric: 'user' }).$}
+          Icon={LeadershipIcon}
```

Comment:
The icon is not the same as the Leadership one

![Screenshot 2024-04-01 at 13 26 05](https://github.com/yldio/asap-hub/assets/16595804/e995777d-f265-4a05-94a7-00316b11527f)